functions/src/models/Phone.ts

```diff
@@ -0,0 +1,84 @@
+import * as admin from 'firebase-admin';
+import * as functions from 'firebase-functions';
+import { getParticipantRef } from './Participant';
+
+export type VerificationStatus = 'pending' | 'approved' | 'canceled';
+export const getVerificationStatus = (statusStr: VerificationStatus | undefined): VerificationStatus | undefined => {
+  if (!statusStr) return undefined;
+  if (new Set<VerificationStatus>(['pending', 'approved', 'canceled']).has(statusStr)) return statusStr;
+
+  return undefined;
+};
```

Comment:
This reads super weird. Would love to know a better (strongly-typed) version of this.